cmd/aries-js-worker/src/agent-rest-client.js

```diff
@@ -448,6 +448,12 @@ const pkgs = {
             method: "POST",
         },
     },
+    context: {
+        AddContext: {
+            path: "/jsonld/context/add",
```

Comment:
is this the usual pattern for add?

Reply:
'/add' suffix is unusual for a REST API endpoint. The HTTP POST verb denotes its save/add ie, `HTTP POST /jsonld/context`